package.json

```diff
@@ -52,7 +52,7 @@
     "postcss-inline-svg": "^6.0.0",
     "postcss-modules": "^6.0.0",
     "prettier": "^2.7.1",
-    "sass": "^1.49.0",
+    "sass": "^1.77.8",
```

Comment:
Upgrading sass is needed, because our older version does not contain the warning